nixos/modules/services/hardware/undervolt.nix

```diff
@@ -25,19 +35,23 @@ let
     uncore = cfg.uncoreOffset;
     analogio = cfg.analogioOffset;
 
+    temp = cfg.temp;
     temp-bat = cfg.tempBat;
-    temp-ac = cfg.tempAc;
 
     power-limit-long = mkPLimit cfg.p1.limit cfg.p1.window;
     power-limit-short = mkPLimit cfg.p2.limit cfg.p2.window;
   };
 in
 {
+  imports = [
+    (lib.mkRemovedOptionModule [ "options" "services" "undervolt" "tempAc" ] "Use options.services.undervolt.temp instead")
+  ];
+
   options.services.undervolt = {
     enable = mkEnableOption ''
-       Undervolting service for Intel CPUs.
+      Undervolting service for Intel CPUs.
 
-       Warning: This service is not endorsed by Intel and may permanently damage your hardware. Use at your own risk
+      Warning: This service is not endorsed by Intel and may permanently damage your hardware. Use at your own risk!
```

Comment:
nit: period is appended here when using `mkEnableOption`, so this should be reworded to not have `!.` at the end, or use `mkOption` itself.